app/packages/state/src/hooks/useCreateLooker.ts

```diff
@@ -105,24 +107,39 @@ export default <T extends AbstractLooker<BaseState>>(
         const filePath =
           urls.filepath?.split("?")[0] ?? (sample.filepath as string);
 
-        if (filePath.endsWith(".pcd") || filePath.endsWith(".fo3d")) {
-          create = ThreeDLooker;
-        } else if (mimeType !== null) {
-          const isVideo = mimeType.startsWith("video/");
+        // media types for which we have a dedicated looker
+        const nativeMediaTypes = [
+          // undefined media_type is assumed to be an image
+          undefined,
+          "image",
+          "video",
+          "3d",
+          "point-cloud",
+          "group",
+        ];
```

Comment:
Maybe this is an opportunity to break out at least the constructor selection logic to a new function? `useCreateLooker` is terribly bloated and nearly untestable at this point 😅